### PR TITLE
[FIX] Acquire public IP address does not give IP selection option

### PR DESCRIPTION
**Pull request by utchoang**
Open for https://github.com/apache/cloudstack-primate/issues/392
@rhtyd cc @svenvogel I've completed this task. Please review and let me know any comments.
image

### CODE REVIEW

**Comment by rhtyd** [comment]
@blueorangutan package

File: src/views/network/IpAddressesTab.vue [hunk 1]
```diff
@@ -181,6 +214,21 @@ export default {
...
+        const params = {
+          zoneid: this.resource.zoneid,
+          domainid: this.resource.domainid,
+          account: this.resource.account,
+          forvirtualnetwork: true,
+          allocatedonly: false
+        }
+        api('listPublicIpAddresses', params).then(json => {
+          const listPublicIps = json.listpublicipaddressesresponse.publicipaddress || []
+          resolve(listPublicIps)
```
Comment (rhtyd) [inline src/views/network/IpAddressesTab.vue:228]: Can you sort the IPs @utchoang (and make the list searching, like other a-selects)?

Comment (utchoang) [reply src/views/network/IpAddressesTab.vue:229]: @rhtyd I've fixed it.

**Comment by rhtyd** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@utchoang can you check why this call the API twice? In my testing, it was calling the API twice: (I've not reviewed code yet)
```
zoneid: b6afade4-1919-4f0b-a785-12ff05a4d383
domainid: e4874e10-5fdf-11ea-9a56-1e006800018c
account: rohit
forvirtualnetwork: true
allocatedonly: false
command: listPublicIpAddresses
response: json
```

**Comment by utchoang** [comment]
@rhtyd Sorry about this. I forgot to delete the test call function, so I called calling API twice. I've removed it. Thanks.

**Comment by rhtyd** [comment]
@blueorangutan package

**Pull request merged**